package.json

```diff
@@ -15,6 +15,7 @@
   },
   "files": [
     "transform.js",
```

Comment:
Shouldn't we remove this one. We don't have this file.

Reply:
you are correct!